packages/ui/test/working-groups/model/getRewardAmount.test.ts

```diff
@@ -0,0 +1,18 @@
+import BN from 'bn.js'
+
+import { getReward } from '@/working-groups/model/getReward'
+
+describe('getReward', () => {
+  it('Example use', () => {
+    const reward = getReward(2, 'forum')
+    expect(reward.payout.toNumber()).toEqual(28820)
+    expect(reward.blockInterval).toEqual(14410)
+  })
+
+  it('Not a known group name', () => {
+    expect(getReward(1, 'incorrect')).toEqual({
+      payout: new BN(0),
+      blockInterval: 14400,
+    })
```

Comment:
The interval is not known, so the fallback should return:

```suggestion
    expect(getReward(7, 'incorrect')).toEqual({
      payout: new BN(7),
      blockInterval: 1,
    })
```

as we know, that we pay 7 per block (still not correct, but IMO better than `0` per something.